src/main/java/TaskManager.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Arrays;
+
+public class TaskManager {
+
+    public TaskManager(){
+        // TODO: 02-Feb-21
+    }
+
+    private static Task[] tasksObjectsArray = new Task[100];
```

Comment:
Maybe it will be better to use a CONSTANT variable for the value 100.  This would help readers understand the significance of the value :)

Reply:
Thanks for the feedback, I have made the changes :)